test/e2e/air_gapped_test.go

```diff
@@ -34,7 +34,7 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-var _ = Describe("Install Controlplane, Core, Bootstrap Providers in an air-gapped environment", func() {
+var _ = Describe("Install Controlplane, Core, Bootstrap Providers in an air-gapped environment", Ordered, func() {
```

Comment:
```suggestion
var _ = Describe("Install Controlplane, Core, Bootstrap Providers in an air-gapped environment", func() {
```

What is the need for `Ordered` in here? These tests should be capable of running in parallel in any order, but if they need a prerequisite steps - please add it to the test case itself.

Reply:
Seems to tests are not designed to work independently. I fixed this bug and tests are failed without fixing order. I will try till make them independent or combine most of pipelines in the single It in next PR.